core/src/test/scala/integration/kafka/server/KRaftClusterTest.scala

```diff
@@ -437,18 +438,6 @@ class KRaftClusterTest {
     }
   }
 
-  @Test
-  def testCreateClusterInvalidMetadataVersion(): Unit = {
```

Comment:
It's no longer possible to set the bootstrap metadata version to lower than 3.3-IV3.